### ISSUE TITLE
Adapt CoaddObs to allow stack inputs

### ISSUE THREAD

**Issue by esheldon** [LSSTDESC/descwl_coadd#41]
Currently it expects galsim wcs objects as input

Also, what we get from the stack is the `skyInfo` which has the stack coadd wcs and a bounding box.  But `CoaddObs` gets the dimensions of the coadd and makes its own boxes

Finally, remove exposures that have edges in the coadd region

**Comment by esheldon** [comment]
Need to have this flag defined somewhere other than the sim code

https://github.com/LSSTDESC/descwl_coadd/blob/master/descwl_coadd/coadd.py#L28

**Comment by esheldon** [comment]
This goes back to the need for the simulation code to produce stack exposures rather than these `SEObs` we have.

It is important to  not have two different code paths for simulated and real data

**Comment by beckermr** [comment]
Yup. The plan was to write a converter or add a method that would convert the SEObs to the stack format. We could ofc just work directly with the stack data structures too.

**Comment by esheldon** [comment]
I think it makes sense at this point to create exposures in the sim code, unless there is something we cannot do with the stack primitives.

This is going to be significant work, since I have no idea how to do a lot of things we need to do.
